docs/reference/concepts/02-provider.mdx

```diff
@@ -110,31 +146,31 @@ export class MyFeatureProvider implements Provider {
   resolveBooleanEvaluation(
     flagKey: string,
     defaultValue: boolean,
-    context: EvaluationContext
+    context: EvaluationContext,
   ): Promise<ResolutionDetails<boolean>> {
     // code to resolve boolean details
   }
 
   resolveStringEvaluation(
     flagKey: string,
     defaultValue: string,
-    context: EvaluationContext
+    context: EvaluationContext,
```

Comment:
Any particular reason we added the trailing comma for the last parameter?

Reply:
I think my IDE formatting just did that automatically.

Reply:
Most of our codebase uses a trailing comma.